egs/librimix/ConvTasNet/eval.py

```diff
@@ -0,0 +1,146 @@
+import os
+import random
+import soundfile as sf
+import torch
+import yaml
+import json
+import argparse
+import pandas as pd
+from tqdm import tqdm
+from pprint import pprint
+import numpy as np
+
+from asteroid.metrics import get_metrics
+from asteroid.data.librimix_dataset import LibriMix
+from asteroid.losses import PITLossWrapper, pairwise_neg_sisdr
+from asteroid.torch_utils import load_state_dict_in
+from asteroid.utils import tensors_to_device
+
+from model import make_model_and_optimizer
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--test_dir', type=str, required=True,
+                    help='Test directory including the json files')
+parser.add_argument('--task', type=str, required=True,
+                    help='One of `enh_single`, `enh_both`, '
+                         '`sep_clean` or `sep_noisy`')
+parser.add_argument('--out_dir', type=str, required=True,
+                    help='Directory in exp_dir where the va will be stored')
+parser.add_argument('--use_gpu', type=int, default=0,
+                    help='Whether to use the GPU for model execution')
+parser.add_argument('--exp_dir', default='exp/tmp',
+                    help='Experiment root')
+parser.add_argument('--n_save_ex', type=int, default=10,
+                    help='Number of audio examples to save, -1 means all')
+
+compute_metrics = ['si_sdr', 'sdr', 'sir', 'sar', 'stoi']
+
+
+def main(conf):
+    # Make the model
+    model, _ = make_model_and_optimizer(conf['train_conf'])
+    # Load best model
+    with open(os.path.join(conf['exp_dir'], 'best_k_models.json'), "r") as f:
+        best_k = json.load(f)
+    best_model_path = min(best_k, key=best_k.get)
+    # Load checkpoint
+    checkpoint = torch.load(best_model_path, map_location='cpu')
+    state = checkpoint['state_dict']
+    state_copy = state.copy()
+    # Remove unwanted keys
+    for keys, values in state.items():
+        if keys.startswith('loss'):
+            del state_copy[keys]
+            print(keys)
+    model = load_state_dict_in(state_copy, model)
```

Comment:
We should probably add this logic to the usual loading function as the loss func is never in the model. Keep in mind for a future PR

Reply:
Should I remove it from the script and use the usual load_best_model_function ?

Reply:
No, I think it's fine for now. 
I just commented so that I'll remember